Source/Shared/BPConfiguration.h

```diff
@@ -51,6 +54,7 @@
 @property (nonatomic, strong) NSNumber *numSims;
 @property (nonatomic) BOOL listTestsOnly;
 @property (nonatomic) BOOL quiet;
+@property (nonatomic) int program; // one of BLUEPILL or BP
```

Comment:
yah, I think that is better than `#define BP 2`

```
typedef NS_ENUM(NSInteger, BPProgram) {
    BP = 1,
    BLUEPILL = 2,
};
```

Both = BP | BLUEPILL, how do you think